### ISSUE TITLE
Angular degree search - wp.a11y prints live regions at bottom of page, not included in landmark region

### ISSUE THREAD

**Issue by cjg89** [UCF/UCF-Degree-Search-Plugin#109]
**Describe the bug**

`wp.a11y.speak()`, which we use to announce changes to degree search results when the user applies filters or searches for something, prints its markup right above the closing `</body>` tag wherever it's used.  We get dinged for this in accessibility scans because these live regions don't live within a landmark element.

**Steps to Reproduce**

View accessibility scans for /degree-search/.

**Expected Behavior**

To resolve this issue, all "perceivable text content" should be included with an ARIA landmark.

**Possible Fix**

One way we could fix this would be by making our existing results message wrapper a live region, by adding `aria-live="polite"` to this `<p>`: https://github.com/UCF/UCF-Degree-Search-Plugin/blob/abfcc55c27ddd1a2cdf6075f2ec6ec74ff2504b4/includes/ucf-degree-search-angular-common.php#L274-L283 and [dequeuing](https://github.com/UCF/UCF-Degree-Search-Plugin/blob/abfcc55c27ddd1a2cdf6075f2ec6ec74ff2504b4/includes/ucf-degree-search-angular-common.php#L138) + removing existing logic in the degree search js that calls `wp.a11y.speak()`.  May need to make slight adjustments to how "no results found" messages are rendered on screen to ensure this content is updated and announced consistently.

**Your environment:**

n/a

**Additional context**

n/a
